flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/ChangelogKeyedStateBackend.java

```diff
@@ -524,22 +496,6 @@ public void notifyCheckpointAborted(long checkpointId) throws Exception {
             // This might change if the log ownership changes (the method won't likely be needed).
             stateChangelogWriter.reset(lastUploadedFrom, lastUploadedTo);
         }
```

Comment:
I think it deserved to add a `TODO` to implement notfiy the delegated state backend at proper time.

Reply:
Good point.
I've created FLINK-25842, will mention that ticket in the comment.